Core/Model/Subcuenta.php

```diff
@@ -184,12 +184,30 @@ public function getIdAccount(): int
             new DataBaseWhere('codejercicio', $this->codejercicio),
             new DataBaseWhere('codcuenta', $this->codcuenta),
         ];
-
         $account = new Cuenta();
         $account->loadFromCode('', $where);
+
         return $account->idcuenta;
     }
 
+    /**
+     * Load de ID for subAccount
+     *
+     * @return int
+     */
+    public function getIdSubaccount(): int
+    {
+        $where = [
+            new DataBaseWhere('codejercicio', $this->codejercicio),
+            new DataBaseWhere('codsubcuenta', $this->codsubcuenta),
+        ];
+        $subaccount = new Subcuenta();
+        $subaccount = $subaccount->all($where);
+
+        return $subaccount[0]->idsubcuenta;
```

Comment:
This is a bad implementation. What happens if there are no records?

This is better:
$subaccount = new Subcuenta();
foreach($subaccount->all($where) as $subc) {
   return $subc->idsubcuenta;
}

return 0;